src/Header.css

```diff
@@ -6,6 +6,7 @@ nav {
   align-items: center;
   width: 100%;
   background-color: slategrey;
+  margin-bottom: 30px;
```

Comment:
Is there a need for margin-bottom on the header? I cannot see any changes when removing it.

Reply:
Okay I m removing that